src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -28,17 +28,17 @@
 
 helps['acr replication'] = """
     type: group
-    short-summary: Manage replications for Azure Container Registries.
+    short-summary: Manage replications of Azure Container Registries across multiple regions
     """
 
 helps['acr build-task'] = """
     type: group
-    short-summary: Manage build tasks for Azure Container Registries.
+    short-summary: Manage a build definitions, which can be triggered by git commits or base image updates.
```

Comment:
Are we using the words "build definition" instead of "build task" in the description of commands you just updated. I see that in a few places below as well and looks inconsistent.

Also, grammar (Manage a build task)?

Reply:
repeating the name build task in the definition doesn't really help. I'm using a build definition to be the definition of a build task